### ISSUE TITLE
Why is the accuracy all zero when training

### ISSUE THREAD

**Issue by czy9987** [ybarancan/STSU#8]
I trained for nearly 20 cycles, but the accuracy was always zero

**Comment by ybarancan** [comment]
Which accuracy? There is no metric called accuracy. Are the training metrics or test metrics zero? Is training loss decreasing? Have you checked the visuals? How are you tracking the ''accuracy'', from the console logs or txt files? Please provide some informaiton when you raise an issue.

**Comment by czy9987** [comment]
![image](https://user-images.githubusercontent.com/63067723/184650484-0b0dc933-47ab-4a0a-98ea-fddaa488a8f4.png)
Sorry, I didn't describe my meaning clearly before. I loaded the recommended backbone, but the verification results are always as follows: all indicators are zero

**Comment by ybarancan** [comment]
How are the training loss and training metrics?

**Comment by czy9987** [comment]
The loss is relatively normal, but the training metrics is not ideal

**Comment by czy9987** [comment]
EPOCH : 15
TRAIN：
ERROR:root:LOSS loss_refine : 0.1088651
ERROR:root:LOSS loss_ce : 0.11835777
ERROR:root:LOSS loss_bbox : 0.11141512
ERROR:root:LOSS loss_polyline : 0.1793564
ERROR:root:LOSS loss_endpoints : 0.15135404
ERROR:root:LOSS loss_assoc : 0.33669746
ERROR:root:LOSS loss_end_match : 0.036952976
ERROR:root:LOSS loss_obj_ce : 0.27029768
ERROR:root:LOSS loss_obj_center : 0.08994838342828265
ERROR:root:LOSS loss_obj_len : 0.011928739501715843
ERROR:root:LOSS loss_obj_orient : 0.29827932
ERROR:root:MEAN F : 0.5410993480724764
ERROR:root:ASSOC F : 0.30719323376674346
ERROR:root:DETECTION : 0.49998529455016033
ERROR:root:RAW OBJECT MIOU : [[0.         0.         0.         0.        ]
 [0.         0.         0.         0.        ]
 [0.07142856 0.06762028 0.08079624 0.08137044]
 [0.         0.         0.         0.        ]
 [0.         0.         0.         0.        ]
 [0.         0.         0.         0.        ]
 [0.         0.         0.         0.        ]
 [0.         0.         0.         0.        ]]
ERROR:root:REFINED OBJECT MIOU : [0.         0.         0.4255813  0.         0.         0.09523806
 0.         0.        ]

**Comment by ybarancan** [comment]
Firstly, do you get any errors in confusion update? Line 317 - 323 . Secondly, do you see the message  logging.error('DETECTED NOTHING') in validation?
